### PR TITLE
[wasm][doc] Add Packager doc

### PR DESCRIPTION
**Pull request by kjpou1**

### CODE REVIEW

File: sdks/wasm/docs/getting-started/packager.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# WebAssembly packager.exe
+
+`Packager.exe` is an interim utility for publishing Mono WebAssembly projects while additional tooling is being developed. 
+
```
Comment (vargaz) [inline sdks/wasm/docs/getting-started/packager.md]: This should be 'packager.exe'

File: sdks/wasm/docs/getting-started/packager.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+Access to mono sdk which is controlled by:
+
+```
+-mono-sdkdir=
+```
+
+Steps of execution are:
+
+1. Setup options passed from the command line.
```
Comment (vargaz) [inline sdks/wasm/docs/getting-started/packager.md]: Not sure this needs to be in a 'getting started' document.

File: sdks/wasm/docs/getting-started/packager.md (hunk 1 above)
Comment (kjpou1) [reply sdks/wasm/docs/getting-started/packager.md]: Addressed in recent commit

File: sdks/wasm/docs/getting-started/packager.md (hunk 2 above)
Comment (kjpou1) [reply sdks/wasm/docs/getting-started/packager.md]: I was wondering the same after I already committed it.  I will move it directly to the `docs` directory

Comment (kjpou1) [reply sdks/wasm/docs/getting-started/packager.md]: Addressed in latest commit.  Moved to docs directory and a link to the main WebAssembly readme

File: sdks/wasm/docs/packager.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
+# WebAssembly packager.exe
+
+`packager.exe` is an interim utility for publishing Mono WebAssembly projects while additional tooling is being developed. 
+
+The command line arguments for Packager.exe can be displayed by using the help command line option.
+
+``` bash
+packager.exe —help
```
Comment (kg) [inline sdks/wasm/docs/packager.md]: emdash instead of --

Comment (kjpou1) [reply sdks/wasm/docs/packager.md]: Addressed in recent commit

**Comment by kjpou1** [comment]
@monojenkins build failed

**Pull request merged**